### DISCUSSION TITLE
Sort and filter BP list

### DISCUSSION THREAD

**Discussion by jdheeter** [eoscostarica/eos-rate#793] [Ideas]
On this page: https://eosrate.io/block-producers

I'd like to be able to sort the list based on rankings or number of ratings.
Also I'd like to filter, like to only see the BPs I have/haven't voted for.

**Comment by xavier506** [comment]
Hello @jdheeter  , thanks for the feedback!

Have you seen the sorting option in the menu bar ? 

![image](https://user-images.githubusercontent.com/5632966/134734373-e301831c-fcde-4270-b6e3-23ab2f03b516.png)

Maybe this is an opportunity to improve the sorting / filtering experience?

@murillojorge is our UX guru, any thoughts?

**Reply by jdheeter** [reply]
Ah, I totally did not see that, On desktop the side menu is closed by default. I was expecting the filtering/sorting to be at the top of the page. Maybe make the side menu open by default on desktop, and on mobile add a sort section at the top of the page so people won't miss it.

**Reply by murillojorge** [reply]
Yeah there's definitely some lostness there. We'll fix it. Thanks for spotting it.

**Comment by murillojorge** [comment]
Check this out. We move the filters to the top. I'd also like to include a small title at the top. What do you guys think?
https://zpl.io/b6wzQ0W

![Sorting - Desktop](https://user-images.githubusercontent.com/1179619/134748262-44f46859-9b04-42e3-af97-bad87172decc.png)

**Reply by jdheeter** [reply]
Looks good, I would actually give that top area more padding, maybe a different color, and have labels above the dropdown buttons like 

sort: filter:

**Comment by xavier506** [comment]
we should enable sorting for proxies as well 

see #745
